test/js/ez-navigation-hub.js

```diff
@@ -7,4 +12,128 @@ describe('ez-navigation-hub', function() {
+                assert.equal(element.activeZoneClass, value);
+            });
+        });
+
+        describe('`matchedLinkClass`', function () {
+            it('should have a default value', function () {
+                assert.equal(element.matchedLinkClass, element.getAttribute('matched-link-class'));
+            });
+
+            it('should set', function () {
+                const value = 'Something';
+
+                element.matchedLinkClass = value;
+                assert.equal(element.matchedLinkClass, value);
+            });
+        });
+
+        describe('`activeZone`', function () {
+            it('should default to undefined', function () {
+                assert.isUndefined(element.activeZone);
+            });
+
+            it('should set', function () {
+                const value = 'contents';
+
+                element.activeZone = value;
+                assert.equal(element.activeZone, value);
+            });
+        });
+
+        describe('`matchedLinkUrl`', function () {
+            it('should default to undefined', function () {
+                assert.isUndefined(element.matchedLinkUrl);
+            });
+
+            it('should set', function () {
+                const value = 'url1';
+
+                element.matchedLinkUrl = value;
+                assert.equal(element.matchedLinkUrl, value);
+            });
+        });
+    });
+    describe('active zone update', function () {
+        function simulateClick(element) {
+            const click = new CustomEvent('click', {
+                bubbles: true,
+                cancelable: true,
+            });
+
+            element.dispatchEvent(click);
+            return click;
+        }
+        function assertEventHandled(event) {
+            assert.ok(
+                event.defaultPrevented,
+                'The event should have been prevented'
+            );
+        }
+        describe('highlight zone', function () {
+            it('should highlight zone contents', function () {
+                const zone = element.querySelector('[data-zone-identifier="contents"]');
+                const link = zone.querySelector('a');
+                const event = simulateClick(link);
+
+                assertEventHandled(event);
+                assert.isOk(zone.classList.contains(element.activeZoneClass));
+            });
+
+            it('should highlight zone other and unhighlight zone contents', function () {
+                const contentsZone = element.querySelector('[data-zone-identifier="contents"]');
+                const otherZone = element.querySelector('[data-zone-identifier="other"]');
+                const link = otherZone.querySelector('a');
+
+                element.activeZone = 'contents';
+                const event = simulateClick(link);
+                assertEventHandled(event);
+
+                assert.isNotOk(contentsZone.classList.contains(element.activeZoneClass), 'zone contents should not be highlighted');
```

Comment:
the assert message is confusing, please put name coming from fixture (`contents` here, `other` below) between quotes or back tick to ease readability